blog/_posts/2021-03-19-tmva-pytorch.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "TMVA PyTorch Interface is Out!"
+layout: archive
+author: Anirudh Dagar
+---
+
+What if we combine [PyTorch][PyTorch] & [TMVA][TMVA]? Ever wondered how ROOT utilizes powerful external MVA libraries making them easily accessible with a direct integration into the TMVA workflow? These interfaces between TMVA & Python frameworks are powered by the PyMVA backbone. All PyMVA methods provide the same plug-and-play mechanisms as TMVA.
+
+With the release of *ROOT v6-24-00* we are excited to launch a brand new PyTorch Interface for TMVA.
+
+[PyTorch][PyTorch] is a Python-based scientific package supporting a​utomatic differentiation.​ An ​open-source machine learning​ framework that accelerates the path from research prototyping to production deployment.
+
+[PyTorch]: https://pytorch.org/
+[TMVA]: https://root.cern/manual/tmva/
+
+## Need for a PyTorch Interface?
+
+TMVA already has a PyKeras interface which we all love, especially with Keras’ simple high-level tensorflow API. If your work involves some elementary experiments, Keras maybe the goto framework due to its plug & play spirit.
+
+But things get interesting when one requires low level control and flexibility. That’s when the argument for Keras starts losing water. PyTorch on the other hand is amazing in terms of control, flexibility and raw power that it can provide to the user. It’s lower-level approach is better suited for the more mathematically-inclined users.
+
+PyTorch is widely used​ among researchers and hence has a large community around it.
+
+* ROOT + PyTorch: Allows to integrate ROOT methods which are good at handling HEP data and PyTorch for Machine Learning.
+
+* Power & Flexibility: Neural Nets are not easy to develop using TMVA, as they require complex configuration strings. Even with PyKeras Interface, designing custom layers is not feasible. PyTorch offers the power and flexibility to achieve complex models with custom layers, optimizers, loss functions and training methodologies.
+
+* Ease of Debugging: ​PyTorch models make use of dynamic computation graphs and are based on eager execution. This makes it easier to use debugging tools like pdb.
+
+* Performance: PyTorch is extremely fast due to it’s highly optimized C++ backend.
+
+
+Designing a simple neural net in PyTorch using a [PyTorch container][Containers] is extremely simple. Here we use [`nn.Sequential`][Sequential]:
```

Comment:
"Neural Net" was capitalized before. I would never capitalize it, but anyway we should be consistent

Reply:
I capitalized it because it was at the start of the sentence. Anyways replaced it with "model", should be good to go.